udf/get_user_prefs.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TEMP FUNCTION udf_get_user_prefs(user_prefs STRING)
+RETURNS STRUCT<user_pref_browser_launcherprocess_enabled BOOLEAN,
+  user_pref_browser_search_widget_innavbar BOOLEAN,
+  user_pref_browser_search_region STRING,
+  user_pref_extensions_allow_non_mpc_extensions BOOLEAN,
+  user_pref_extensions_legacy_enabled BOOLEAN,
+  user_pref_gfx_webrender_all_qualified BOOLEAN,
+  user_pref_marionette_enabled BOOLEAN,
+  user_pref_privacy_fuzzyfox_enabled BOOLEAN,
+  user_pref_dom_ipc_plugins_sandbox_level_flash INT64,
+  user_pref_dom_ipc_processcount INT64,
+  user_pref_general_config_filename STRING,
+  user_pref_security_enterprise_roots_auto_enabled BOOLEAN,
+  user_pref_security_enterprise_roots_enabled BOOLEAN,
+  user_pref_security_pki_mitm_detected BOOLEAN,
+  user_pref_network_trr_mode INT64> AS ((
+  SELECT AS STRUCT
```

Comment:
any time here where we use `(SELECT AS STRUCT ...)` we should instead use `STRUCT(...)` if possible

Reply:
fixed